app/Resources/translations/messages.sv.xliff

```diff
@@ -518,6 +518,31 @@
                     Best regards,
                     Kompisbyrån</target>
             </trans-unit>
+            <trans-unit id="match.email.user">
+                <source>match.email.user</source>
+                <target>Hello %user_name%!
+
+Great that you want to be part of Kompisbyrån!
```

Comment:
Then set the locale parameter i parameters.yml to en. Texts should be in place when merging. 

On fre 15 apr. 2016 at 08:50 Mikael

<
mailto:Mikael <notifications@github.com>
> wrote:

In
https://github.com/kompisbyran/Kompisbyran/pull/86#discussion_r59833147
:

> @@ -518,6 +518,31 @@ > Best regards, > Kompisbyrån</target> > </trans-unit> > + <trans-unit id="match.email.user"> > + <source>match.email.user</source> > + <target>Hello %user_name%! > + > +Great that you want to be part of Kompisbyrån!

https://github.com/jongotlin
I will do this right away after it is deployed. We will do the trial with english for now and use the old system until we have the swedish in place (it is already written)

—

You are receiving this because you were mentioned.

Reply to this email directly or
https://github.com/kompisbyran/Kompisbyran/pull/86/files/e7a28ca774f536331ce134a753f11db63ce223d3#r59833147
https://github.com/kompisbyran/Kompisbyran/pull/86/files/e7a28ca774f536331ce134a753f11db63ce223d3#r59833147